packages/grid/test/row-details.test.js

```diff
@@ -46,7 +46,7 @@ describe('row details', () => {
     grid.rowDetailsRenderer = simpleDetailsRenderer;
     grid.querySelector('vaadin-grid-column').renderer = indexRenderer;
 
-    const spy = sinon.spy(grid, '_updateRow');
+    const spy = sinon.spy(grid, '__initRow');
```

Comment:
Could probably rename test title above accordingly to mention "row init count" instead of "update count".